webclient/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.1.0",
   "description": "",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "install": "napa Amoenus/SwaggerDark#b6fa2536da4e60b3fa15fa5e62eaac668dbc2a21:SwaggerDark"
```

Comment:
Is this necessary?
Wouldn't it be easier to just privately fork `Amoenus/SwaggerDark`, add a `package.json` and handle it like `github:Valexr/spectre`?

Reply:
Doing so could also solve the issue, that [this css file](https://github.com/Amoenus/SwaggerDark/blob/b6fa2536da4e60b3fa15fa5e62eaac668dbc2a21/SwaggerDark.css#L1) overrides some values, we ceartainly dont want overridden..

Reply:
![image](https://user-images.githubusercontent.com/26258709/184555468-89a17f6b-bc54-404f-9e8c-043f109fc264.png)

Reply:
I also thought about directly including the css file in the repo, but the fork approach is probably a bit more consistent with the rest of the build.

Reply:
I've created a private fork at https://github.com/octycs/SwaggerDark to not clutter the TUM-Dev namespace